projects/cashmere/src/lib/pagination/base-pagination.ts

```diff
@@ -48,8 +56,10 @@ export class BasePaginationComponent extends Initailizable implements OnInit {
     private _pageSize: number = BasePaginationComponent._DEFAULT_PAGE_SIZE;
 
     /** Event emitted when the paginator changes the page size or page index. */
-    @Output()
-    readonly page: EventEmitter<PageEvent> = new EventEmitter<PageEvent>();
+    @Output() readonly page: EventEmitter<PageEvent> = new EventEmitter<PageEvent>();
+
+    @Output() readonly pageNumberChange: Observable<number> = this.page.pipe(map(e => e.pageNumber));
```

Comment:
These allow two-way data binding to those individual properties, if you are not interested in getting the entire `(page)` event.  See `pagination-simple-example` above.